cdbot/constants.py

```diff
@@ -91,6 +91,36 @@ class Exchange:
     r'(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'  # hyperlinks
 ]
 
+ELITE_ALISES = {
```

Comment:
This feels backwards. Why not have the camps with the aliases and like
```py
{
    "London": ["Lo", "Lon"]
}
```

Have some nice method for fetching them that way.

Reply:
I agree in principle, but that is an O(N^2) solution. What we really need is an automated program/function to convert your pretty way of noting it down to a an efficient hash-table at the beginning of execution.

Reply:
> I agree in principle, but that is an O(N^2) solution

> Using python
> Talking about scalability

Reply:
hello just noticed typo this should be ALIASES not ALISES